lib/adhearsion/script_ahn_loader.rb

```diff
@@ -8,9 +8,22 @@ module ScriptAhnLoader
     RUBY = File.join(*RbConfig::CONFIG.values_at("bindir", "ruby_install_name")) + RbConfig::CONFIG["EXEEXT"]
     SCRIPT_AHN = File.join('script', 'ahn')
 
+    def self.load_script_ahn(path = Dir.pwd)
+      path = Pathname.new(path).expand_path
+      begin
+        script = File.join(path, SCRIPT_AHN)
```

Comment:
It's not so much an implementation detail of that method - that method is an abstraction of what you want to do. I was just looking for ways to make this read more naturally.